.evergreen/config.yml

```diff
@@ -2708,8 +2723,9 @@ buildvariants:
      python-version: ["pypy3.8", "pypy3.10"]
      green-framework: "*"
      auth-ssl: "*"
+     encryption: ["encryption"]
   display_name: "${green-framework} ${python-version} ${platform} ${auth-ssl}"
-  tasks: *all-server-versions
+  tasks: *encryption-server-versions
```

Comment:
This doesn't actually test with encryption since it's missing the encryption tag. How about we open a new ticket to add this test coverage rather than add it here?

Reply:
https://jira.mongodb.org/browse/PYTHON-4296

Reply:
Removed